src/Create2Factory.ts

```diff
@@ -54,7 +54,7 @@ export class Create2Factory {
         21000
 
       // deployer requires some extra gas
-      gasLimit = Math.floor(gasLimit * 64 / 63)
+      gasLimit = Math.floor(gasLimit * 65 / 63)
```

Comment:
have no idea why it was broke. I understand that this Create2Factory is problematic (it REQUIRES manual gas and not "estimate", since it doesn't revert on failure). It is now used only in tests. should switch to Arachnid's deterministic deployer

Reply:
I'm not trying to fix it now (the alternative is to put a fixed gas on the tests that use it.